documentation/api/wire_format/report_format_v5.markdown

```diff
@@ -5,6 +5,7 @@ canonical: "/puppetdb/latest/api/wire_format/report_format_v5.html"
 ---
 
 [puppetreportformat]: http://docs.puppetlabs.com/puppet/latest/reference/format_report.html
+[reportsv4]: ../query/v4/reports.markdown
```

Comment:
Are you sure? This is working for me on the master branch: https://docs.puppetlabs.com/puppetdb/master/api/query/v4/reports.html